README.md

```diff
@@ -20,6 +20,8 @@ To prepare your system to deploy Clearwater on Docker, run:
 
     # Checkout clearwater-docker.
     git clone --recursive git@github.com:Metaswitch/clearwater-docker.git
+    
+Edit clearwater-docker/.env so that PUBLIC_IP is set to an IP address that can be used by SIP clients to access the docker host.   E.g. if you are running in AWS, this wants to be the public IP of your AWS VM.
```

Comment:
Maybe worth noting that this appears in Record-Route headers and calls will drop after 30s if this is wrong?